src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileTitleBar.java

```diff
@@ -138,6 +127,17 @@ public void configure(
     );
   }
 
+  public void configureActionMenu() {
+    actionMenuWidget.setActionListener(
+      Action.ADD_TO_DOWNLOAD_CART,
+      (action, e) -> onAddToDownloadList()
+    );
+    actionMenuWidget.setActionListener(
+      Action.SHOW_PROGRAMMATIC_OPTIONS,
+      (action, e) -> onProgrammaticDownloadOptions()
+    );
+  }
```

Comment:
Since the AddToDownloadList and ProgrammaticDownloadOptions widgets are configured here, we need to configure the action listeners here.